client/src/views/Parts.vue

```diff
@@ -55,7 +55,7 @@ export default {
       addtocart(data) {
         this.test = data
         console.log(data)
-        store.commit('addToCart', data)
+        this.$store.commit('addToCart', data)
```

Comment:
The store can be accessed on the Vue component through `$store`. https://vuex.vuejs.org/guide/state.html